tests/fixtures/git/github.com/demo/demo/demo.egg-info/PKG-INFO

```diff
@@ -6,7 +6,9 @@ Home-page: https://github.com/demo/demo
 Author: Sébastien Eustace
 Author-email: sebastien@eustace.io
 License: MIT
-Description: UNKNOWN
 Platform: UNKNOWN
-Provides-Extra: bar
 Provides-Extra: foo
+Provides-Extra: bar
+
+UNKNOWN
```

Comment:
Some test suite instability is causing changes to fixtures that came along for the ride. We need to back these changes out.